test/cases/api.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/bash
+
+#
+# Test osbuild-composer's main API endpoint by building a sample image and
```

Comment:
This comment is incorrect, let's just remove it.

Reply:
Corrected the introduction.